hkmc2DiffTests/src/test/scala/hkmc2/Watcher.scala

```diff
@@ -73,7 +84,7 @@ class Watcher(dirs: Ls[File]):
         completionTime(event.path) = LocalDateTime.now()
       catch ex =>
         // System.err.println("Unexpected error in watcher: " + ex)
-        // ex.printStackTrace()
+        ex.printStackTrace()
```

Comment:
Isn't the stack trace already shown when `throw ex` propagates to the top-level?